docs/pages/components/tables.vue

```diff
@@ -77,6 +77,27 @@ In order to use the table, you need give `rows` and `columns` data to `KtTable`
 
 </ShowCase>
 
+## Lot of columns 
+
+Some table with a lot of columns will need to handle that overflow.
```

Comment:
Tables with lots of columns will need to handle that overflow. This can be accomplished by using the `isScrollable` prop in order to make the table scroll horizontally.